src/test/groovy/ee/tuleva/onboarding/auth/jwt/JwtAuthorizationFilterSpec.groovy

```diff
@@ -119,4 +127,27 @@ class JwtAuthorizationFilterSpec extends Specification {
     1 * filterChain.doFilter(request, response)
   }
 
+  def "does not accept refresh tokens"() {
```

Comment:
Should also test that it doesn't accept other types of tokens (i.e. "asdfg") - to get more confidence that we _only_ accept access_tokens. For example, right now it would not catch the bug in case we would accidentally assert:

```
if (tokenType == TokenType.REFRESH) {
  return;
}
// accept all other token types
```

Reply:
TokenType is an enum, that field is typed and can only have values ACCESS, REFRESH